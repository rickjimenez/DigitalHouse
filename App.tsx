import React from 'react';
import {QueryClient, QueryClientProvider} from '@tanstack/react-query';
import Routes from './src/Routes';

const queryClient = new QueryClient();

const App = () => (
  <QueryClientProvider client={queryClient}>
    <Routes />
  </QueryClientProvider>
);

export default App;
