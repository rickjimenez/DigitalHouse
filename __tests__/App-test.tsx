import 'react-native';
import {render, screen} from '@testing-library/react-native';
import React from 'react';
import App from '../App';

test('renders correctly', () => {
  render(<App />);

  const app = screen.toJSON();
  expect(app).toMatchSnapshot();
});
