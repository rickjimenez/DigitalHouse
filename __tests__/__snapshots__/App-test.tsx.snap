// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    [
      {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <RNSScreenStack
    style={
      {
        "flex": 1,
      }
    }
  >
    <RNSScreen
      collapsable={false}
      forwardedRef={[Function]}
      gestureResponseDistance={
        {
          "bottom": -1,
          "end": -1,
          "start": -1,
          "top": -1,
        }
      }
      nativeBackButtonDismissalEnabled={false}
      onAppear={[Function]}
      onDisappear={[Function]}
      onDismissed={[Function]}
      onHeaderBackButtonClicked={[Function]}
      onNativeDismissCancelled={[Function]}
      onTransitionProgress={[Function]}
      onWillDisappear={[Function]}
      replaceAnimation="push"
      stackPresentation="push"
      style={
        {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
      swipeDirection="horizontal"
    >
      <RNSScreenStackHeaderConfig
        backButtonInCustomView={false}
        backgroundColor="rgb(255, 255, 255)"
        color="rgb(0, 122, 255)"
        direction="ltr"
        disableBackButtonMenu={false}
        hidden={true}
        hideBackButton={false}
        largeTitleHideShadow={false}
        title="Home"
        titleColor="rgb(28, 28, 30)"
        topInsetEnabled={false}
        translucent={false}
      />
      <View
        accessibilityElementsHidden={false}
        importantForAccessibility="auto"
        style={
          {
            "flex": 1,
            "flexDirection": "column-reverse",
          }
        }
      >
        <View
          style={
            [
              {
                "flex": 1,
              },
              {
                "backgroundColor": "rgb(242, 242, 242)",
              },
              undefined,
            ]
          }
        >
          <RCTSafeAreaView
            style={
              {
                "flex": 1,
                "marginHorizontal": 20,
              }
            }
          >
            <View
              style={
                {
                  "marginBottom": 20,
                }
              }
            >
              <Text
                style={
                  {
                    "fontSize": 20,
                    "fontWeight": "800",
                  }
                }
              >
                Bienvenido de vuelta!
              </Text>
              <Text
                style={
                  {
                    "fontSize": 16,
                    "fontWeight": "600",
                  }
                }
              >
                Ruben Rodriguez
              </Text>
            </View>
            <View>
              <Text
                style={
                  {
                    "color": "#9B9898",
                    "fontSize": 14,
                    "fontWeight": "800",
                    "marginBottom": 10,
                  }
                }
              >
                TUS PUNTOS
              </Text>
              <View
                style={
                  {
                    "alignSelf": "center",
                    "backgroundColor": "#334FFA",
                    "borderRadius": 20,
                    "elevation": 5,
                    "height": 143,
                    "justifyContent": "center",
                    "marginBottom": 20,
                    "padding": 20,
                    "shadowColor": "#000",
                    "shadowOffset": {
                      "height": 2,
                      "width": 0,
                    },
                    "shadowOpacity": 0.25,
                    "shadowRadius": 3.84,
                    "width": 286,
                  }
                }
              >
                <ActivityIndicator
                  animating={true}
                  color="white"
                />
              </View>
            </View>
          </RCTSafeAreaView>
        </View>
      </View>
    </RNSScreen>
  </RNSScreenStack>
</RNCSafeAreaProvider>
`;
